### ISSUE TITLE
How to put scalars in the same row in tensorboard in TF2.0?

### ISSUE THREAD

**Issue by zhangjh915** [tensorflow/tensorboard#2984]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 2.0
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**

I did not see related arguments in the documentation. If there is such function, please let me know. :P

The following code produces a tensorboard with 3 different rows. How to put the first two in one row and the second in the other?

```
with summary_writer.as_default():
    tf.summary.scalar("training_loss", loss, step=batch_index)
    tf.summary.scalar("learning_rate", optimizer.learning_rate, step=batch_index)
with summary_writer.as_default():
    tf.summary.scalar("validation_loss", val_loss, step=batch_index)
```

**Will this change the current api? How?**
Probably
**Who will benefit with this feature?**

**Any Other info.**

**Comment by nfelt** [comment]
@zhangjh915 the categories are based on common prefixes in the tag names, delimited by slashes like a filename.  So e.g. if you emit the summaries with the names `tf.summary.scalar("training/training_loss", ...)` and `tf.summary.scalar("training/validation_loss", ...)` it should do what you want.

You can also make this happen automatically byputting them inside a

```python
with tf.name_scope("training"):
  tf.summary.scalar("training_loss", ...)
  tf.summary.scalar("validation_loss", ...)
```

**Comment by david-ngo-sixense** [comment]
Hello @nfelt ,

Do you know how to display two scalars on the same graph (for tensorflow 2.0) ?
I want to display train_accuracy/valid_accuracy on the same graph. (and train_loss/valid_loss)
Thank you very much.

**Comment by psybuzz** [comment]
Thanks for the question.  Unfortunately, merging 2 line charts with 1 line each into 1 line chart with 2 lines is not currently supported in the Scalars dashboard.  There is a separate, existing feature request for adding this, so I would suggest thumbs-up'ing and following the other thread if you are interested: see https://github.com/tensorflow/tensorboard/issues/597

**Comment by david-ngo-sixense** [comment]
Thanks @psybuzz  for your answer.
But I don't really understand. If I use the function model.fit or model.fit_generator; I can easily display two line charts (train accuracy and validation accuracy) into the same chart (to check overfitting). But when I implement the training loops, I can't make it.
train_log_dir = logs_dir.joinpath('train/')
train_log_dir.mkdir(exist_ok=True)
valid_log_dir = logs_dir.joinpath('valid/')
 valid_log_dir.mkdir(exist_ok=True)
 train_summary_writer = tf.summary.create_file_writer(str(train_log_dir))
 valid_summary_writer = tf.summary.create_file_writer(str(valid_log_dir))
  
  with train_summary_writer.as_default():
            with tf.name_scope("accuracy"):
                tf.summary.scalar('train_acc',  train_metrics[0].result(), step=epoch)
   with valid_summary_writer.as_default():
            with tf.name_scope("accuracy"):
                tf.summary.scalar('valid_acc', val_metrics[0].result(), step=epoch)
                
It seems to me that we can do it in tensorflow 1.x

**Comment by psybuzz** [comment]
I see, in the example code, the `train_summary_writer` creates a run with a chart named "train_acc", while the `valid_summary_writer` creates a separate run with a chart named "valid_acc".  If the intent is to compare the same "accuracy" metric across these 2 different runs, does changing the "tag name" of the scalar to the same name address the problem?

e.g.
```python
with train_summary_writer.as_default():
  tf.summary.scalar('accuracy', train_metrics[0].result(), step=epoch)
with valid_summary_writer.as_default():
  tf.summary.scalar('accuracy', val_metrics[0].result(), step=epoch)
```
should produce 1 chart named "accuracy" with 2 lines on it.

**Comment by david-ngo-sixense** [comment]
Yes, it's working @psybuzz . Thanks very much.
